booster_api/features/src/support/planner_jsons/new_import.json

```diff
@@ -0,0 +1 @@
+{"data":{"gitRepository":"test123","projectName":"test123","pipeline":"maven-release","space":"adb3437-d4f4-482e-8459-02ccb0561ea7"}}
```

Comment:
Avoid hardcoding dynamic data like proj name, git repo name, space ID etc. 
You can use a placeholder in your json and then use the replace function to overwrite it (in memory) with real dynamic data.